Update settings>circulation

Lmariomini opened the pull request:
At Molly's request, adding explanatory text to the "For lost items not charged a fee/fine, close the loan after" option.

Lmariomini committed:
Update settings>circulation 

At Molly's request, adding explanatory text to the "For lost items not charged a fee/fine, close the loan after" option.

julianladisch commented on content/en/docs/Settings/Settings_circulation/settings_circulation.md:
```diff
@@ -435,7 +435,7 @@ Lost item fee policies determine when overdue items automatically age to lost (t
 10. Select whether to **Charge lost item processing fee if item declared lost by patron**.
 11. Select whether to **Charge lost item processing fee if item aged to lost by system**.
-12. Enter an amount of time **For lost items not charged a fee/fine, close the loan after**. (This setting is required for Actual cost and does not apply to Set cost.)
+12. Enter an amount of time **For lost items not charged a fee/fine, close the loan after**. This setting is required for Actual cost; it does not apply to Set cost. During the specified time period after an open loan is updated to ‘Declared lost’ or ’Aged to lost,’ Library staff can bill for the item using the [Lost items requiring actual cost](../../../users/#lost-items-requiring-actual-cost) r ...
```
This anchor doesn't exist in Poppy:
`[Lost items requiring actual cost](../../../users/#lost-items-requiring-actual-cost)`
Use 
`[Lost items requiring actual cost](../../../users/#view-lost-items-requiring-actual-cost)`
or
`[Lost items requiring actual cost](../../../users/#processing-lost-items-requiring-actual-cost)`
instead.

Lmariomini replied:
Oops! Thanks for catching this, I was looking at Quesnelia documentation. Should be fixed now!

Lmariomini committed:
Update settings_circulation.md

Explanatory text for "For lost items not charged a fee/fine, close the loan after."
